tokio/src/sync/semaphore.rs

```diff
@@ -333,6 +333,76 @@ use std::sync::Arc;
 /// }
 /// ```
 ///
+/// ## Countdown latch
+///
+/// A `Semaphore` may be used to implement a countdown latch, a synchronization
+/// primitive where one task waits until a set number of other tasks to have
+/// completed before continuing.
+///
+/// The task awaiting the countdown should call [`Semaphore::acquire_many`] to
+/// acquire the a number of permits equal to the number of tasks which hold the
+/// latch open. The tasks whose completion is awaited by the latch should call
+/// [`Semaphore::add_permit`] with a single permit to increment the countdown
+/// latch as they complete. For example:
+///
+/// ```rust
+/// use tokio::sync::Semaphore;
+/// use std::{future::Future, sync::Arc};
```

Comment:
We use this convention in Tokio source code:
```suggestion
/// use std::future::Future;
/// use std::sync::Arc;
```

Reply:
ah, yup, i just copied this from the issue comment and didn't think to change the imports, thanks!